src/util/ActionProgramUtils.js

```diff
@@ -47,6 +49,17 @@ export const makeAaveBorrowAction = async ({
       walletId: borrowEngineWallet.id
     })
 
+  // Construct the Withdraw to Bank action
+  if (isDestBank) {
+    out.push({
+      type: 'fiat-sell',
+      fiatPluginId: 'Wyre', // TODO: update after converting fiatPluginId to displayName
```

Comment:
lower-case this and then change the casing in the display.js file for action-queue.